models/Tweet.js

```diff
@@ -15,36 +27,35 @@ export default class Tweet {
 
   likers(tweet, { lastCreatedAt = 0, limit = 10 }) {
     return this.context.User.collection.find({
-      likedIds: tweet.id,
+      likedIds: tweet._id,
       createdAt: { $gt: lastCreatedAt },
     }).sort({ createdAt: 1 }).limit(limit).toArray();
   }
 
   async insert(doc) {
-    // XXX: proper id generation strategy
-    const id = (await this.collection.find().count()).toString();
     const docToInsert = Object.assign({}, doc, {
-      id,
       createdAt: Date.now(),
       updatedAt: Date.now(),
     });
-    await this.collection.insert(docToInsert);
-    this.pubsub.publish('tweetInserted', docToInsert);
+    const id = (await this.collection.insertOne(docToInsert)).insertedId;
+    this.pubsub.publish('tweetInserted', await this.findOneById(id));
     return id;
   }
 
   async updateById(id, doc) {
```

Comment:
I guess? I don't really like things called `_id` but it does probably make sense for `_id` to be what the model calls it and the resolver to handle the mapping of `_id` to `id`